tutorials/javaFxPart1.md

```diff
@@ -4,6 +4,7 @@
 </frontmatter>
 
 # JavaFX tutorial part 1 – Introduction
+<div style="color:gray; margin:0 0 20px 0; font-size:10px">Last updated: 13/8/2024 </div>
```

Comment:
We can remove 'last updated' from all five pages, as they are a pain to maintain and doesn't really add value.

Reply:
Thank you! Fixed in new commit.